pkgs/by-name/gi/git-of-theseus/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+python3Packages.buildPythonApplication {
+  pname = "git-of-theseus";
+  version = "unstable-2023-11-25";
```

Comment:
This is missing the last releases version before it. When there was none I think we use 0- instead